packages/fx-core/resource/package.nls.json

```diff
@@ -506,6 +506,8 @@
   "error.apiConnector.componentNotExistError": "Component %s not exist, please add first",
   "error.apiConnector.envVarExistError": "Please provide a different API name to avoid conflicts with existing env variables %s in .env.teamsfx.local",
   "error.debugHandler.InvalidSSODebugArgsError": "objectId, clientId and clientSecret are all required if you use existing AAD app for SSO.",
+  "error.debugHandler.InvalidBotDebugArgsError": "botId and botPassword are all required if you use existing bot. botMessagingEndpoint is required.",
```

Comment:
Separate into two errors:
- "botMessagingEndpoint is required."
- "botId and botPassword need to be set together if you use existing bot."

Reply:
Updated.